downloader/src/main/kotlin/VcsHost.kt

```diff
@@ -304,6 +304,18 @@ enum class VcsHost(
                     )
                 }
 
+                ("/tfs/" in projectUrl || ".visualstudio.com" in projectUrl) && "/_git/" in projectUrl -> {
+                    val query = "?version=GB"
+                    val url = projectUrl.substringBefore(query)
+                    val revision = projectUrl.substringAfter(query, "")
+
+                    VcsInfo(
+                        type = VcsType.GIT,
+                        url = url,
+                        revision = revision
+                    )
+                }
```

Comment:
Would be nice to have a test which illustrates how the TFS URLs actually look like.

Reply:
Done.

Reply:
The tests should cover all the conditions under which a TFS URL is detected, also the part with ".visualstudio.com". 
Could it be that the URL contains multiple query parameters? Should the parsing of the version be made more robust, e.g. by splitting the query string at `&` characters and inspecting the key-value pairs?

Reply:
Done.